### PR TITLE
Maints Exit Sign

### PR DESCRIPTION
**Pull request by brainfood1183**
Adds a new diretional exit sign for maints areas which should point to the nearest none departmental airlock.

untitled-411

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Directional Exit signs for maints!

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml [hunk 1]
```diff
@@ -169,6 +169,15 @@
   - type: Sprite
     state: direction_library
 
+- type: entity
+  parent: BaseSignDirectional
+  id: SignDirectionalMaints
+  name: exit sign
+  description: A direction sign, pointing which way to the exit.
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml:176]:
```suggestion
  description: A directional sign pointing toward the nearest exit.
```

**Review by Tayrtahn — APPROVED** [review]

**Review by Tayrtahn — CHANGES_REQUESTED** [review]
Didn't catch this before, but I think it would be better to call this "exit" instead of "maints", for the id, file name, etc.

**Comment by baddiePRO** [comment]
is there any way you could make them glow in the dark too?

**Comment by brainfood1183** [comment]
untitled-412

now glows in dark

**Review by Tayrtahn — APPROVED** [review]

**Comment by Flareguy** [comment]
Due to how wallmounts work, this is going to display on *every side* of a wall, and likely end up being very confusing. This is an interesting idea, but I don't think it can work right now.

**Comment by K-Dynamic** [comment]
Surely we could have some blinking lights on the ground/tile leading to a door

**Comment by UbaserB** [comment]
the glow seems a bit strong

**Comment by SlamBamActionman** [comment]
I support the idea, but perhaps it'd be better to implement it like this:

- Exit signs are mounted on the side of the wall, very similar to lamp tubes.
- Instead of just shining they instead illuminate the surroundings with a small 1-2 tile green glow.

That way they become more diagetic and add to the dark maints atmosphere. Might need a sprite adjustment to work on the side of the wall though.

**Comment by brainfood1183** [comment]
> I support the idea, but perhaps it'd be better to implement it like this:
> 
>     * Exit signs are mounted on the side of the wall, very similar to lamp tubes.
> 
>     * Instead of just shining they instead illuminate the surroundings with a small 1-2 tile green glow.
> 
> 
> That way they become more diagetic and add to the dark maints atmosphere. Might need a sprite adjustment to work on the side of the wall though.

i think this is an interesting idea, kinda like how the janitorial signs work. I will have a think about it.

**Comment by IProduceWidgets** [comment]
> Due to how wallmounts work, this is going to display on _every side_ of a wall, and likely end up being very confusing. This is an interesting idea, but I don't think it can work right now.

This can *maybe* be resolved with careful mapping, but yeah its a bit of a weird spot since we don't want lots of doubled walls.

**Comment by brainfood1183** [comment]
> > Due to how wallmounts work, this is going to display on _every side_ of a wall, and likely end up being very confusing. This is an interesting idea, but I don't think it can work right now.
> 
> This can _maybe_ be resolved with careful mapping, but yeah its a bit of a weird spot since we don't want lots of doubled walls.

Yea currently its best to place signs on T sections of walls to avoid the sign appearing in rooms it shouldn't. Not great lets be honest but it is a problem for all signs and not just this sign i guess.

**Comment by Golinth** [comment]
> > I support the idea, but perhaps it'd be better to implement it like this:
> > ```
> > * Exit signs are mounted on the side of the wall, very similar to lamp tubes.
> > 
> > * Instead of just shining they instead illuminate the surroundings with a small 1-2 tile green glow.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > That way they become more diagetic and add to the dark maints atmosphere. Might need a sprite adjustment to work on the side of the wall though.
> 
> i think this is an interesting idea, kinda like how the janitorial signs work. I will have a think about it.

That’s exactly how I was envisioning this. A glowing exit sign, could even use the same green bulb as the janitorial sign

**Pull request merged**